metaflow/runner/subprocess_manager.py

```diff
@@ -13,13 +13,15 @@
 def kill_process_and_descendants(pid, termination_timeout):
     try:
         subprocess.check_call(["pkill", "-TERM", "-P", str(pid)])
+        subprocess.check_call(["kill", "-TERM", str(pid)])
```

Comment:
there is a race condition here where in between the _pkill_ and _kill_ calls, new child processes can be spawned.